packages/oauth/src/twitch.ts

```diff
@@ -74,22 +74,40 @@ class Twitch<A extends Auth> implements OAuthProvider<A> {
 			})
 		).data[0] as TwitchUser;
 
-		const twitchUserId = String(twitchUser.id);
+		const PROVIDER_ID = "github";
```

Comment:
Looks like this was meant to be `twitch`
```suggestion
		const PROVIDER_ID = "twitch";
```